documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -7,24 +7,36 @@
 = Setting throughput and storage limits on brokers
 
 [role="_abstract"]
-Use the _Kafka Static Quota_ plugin to set throughput and storage limits on brokers in your Kafka cluster.
-You enable the plugin and set limits by configuring the `quotas` section of the `Kafka` resource.
-You can set a byte-rate threshold and storage quotas to put limits on the clients interacting with your brokers.
+This procedure describes how to set throughput and storage limits on brokers in your Kafka cluster.
+Enable a quota plugin and configure limits using `quotas` properties in the `Kafka` resource.
 
-NOTE: Only one quota plugin can be used in Kafka. 
-By enabling this plugin, the built-in Kafka quotas plugin is automatically disabled.
-This means that you won't see per-client quota metrics, but only aggregated quota metrics.
+* The `strimzi` type enables the _Kafka Static Quota_ plugin.
```

Comment:
I see that we removed the "static" word from the repo and plugin names, probably to avoid confusion. Should we simply call it Strimzi Quota plugin?